source/time_tables/departure_boards.h

```diff
@@ -43,10 +43,10 @@ typedef std::pair<boost::optional<routing::datetime_stop_time>, boost::optional<
     first_and_last_stop_time;
 
 void departure_board(PbCreator& pb_creator,
-                     const std::string& filter,
-                     const boost::optional<const std::string> calendar_id,
+                     const std::string& request,
```

Comment:
what? it changes variable's name too?

Reply:
What the name between the declaration and definition don't match yes!